docs/contribute/architecture.md

```diff
@@ -99,3 +99,48 @@ This plugin uses the Node-RED [Authentication API](https://nodered.org/docs/user
 This plugin sends Node-RED Audit events (e.g. user log in and flow deployment events) back to the to the FlowForge Management Application to allow a reliable audit of what actions have taken place in the project.
 
 This plugin uses the Node-RED [Logging API](https://nodered.org/docs/user-guide/runtime/logging)
+
+## Component overview
+
+```mermaid
+erDiagram
+    USER ||--o{ NGINX : Requests
+    NGINX {
+        Protocol HTTP-TLS
+        Port default-443
+    }
+    FORGE-APP {
+        Protocol HTTP-TLS
+        Port default-3000
+    }
+    POSTGRESQL {
+        Protocol tcp-tls
+        Port default-5432
+    }
+    MOSQUITTO {
+        Protocol HTTP-TLS-WSS-MQTT
+        Port default-1883
```

Comment:
```suggestion
        Port default-1883
        Port default-1884
```
The broker runs 2 ports:
- 1883 for native MQTT (used by projects inside the cluster)
- 1884 for MQTT over WS which is proxied by nginx/ingress to port 80/443 outside the cluster